dspace-api/src/main/java/org/dspace/identifier/doi/DOIOrganiser.java

```diff
@@ -227,8 +226,16 @@ public static void runCLI(Context context, DOIOrganiser organiser, String[] args
                 }
 
                 for (DOI doi : dois) {
-                    organiser.reserve(doi);
-                    context.uncacheEntity(doi);
```

Comment:
Why did you removed the `uncacheEntity`?

Reply:
It is replaced by `commit`, which will accomplish the same.